README.md

```diff
@@ -140,7 +140,11 @@ public interface ModelServerClientApiV1<A> {
 
    CompletableFuture<Response<Boolean>> edit(String modelUri, CCommand command, String format);
 
-   void subscribe(String modelUri, SubscriptionListener subscriptionListener, String format);
+   void subscribe(String modelUri, SubscriptionListener subscriptionListener);
```

Comment:
Was it intended to remove the `String format` parameter from the method?

Reply:
Yes, as mentioned in the description of this PR, I removed that argument as it was never offered by the REST API, and only introduced here in the `ModelServerClientApiV1`. But we could open an issue for implementing the feature of enabling subscription on a certain format though.

Reply:
Oops, I didn't read that :) I simply noticed that Ecore GLSP had a compile error with this change.
All good then :+1: